Source/Csla.Blazor/PropertyInfo.cs

```diff
@@ -21,10 +21,6 @@ public class PropertyInfo : IPropertyInfo, INotifyPropertyChanged
     /// Gets the model
     /// </summary>
     protected object Model { get; }
-    /// <summary>
-    /// Gets the property name
-    /// </summary>
-    protected string PropertyName { get; }
```

Comment:
There's no implementation here to actually get and return the property name, so how would this work?

I'd expect an implementation similar to the one in the `FriendlyName` property.

Reply:
Isn't it getting set in the `PropertyInfo` constructor?
https://github.com/MarimerLLC/csla/blob/5ab18303d4a0c9ae218edfcbb6e64e15dd72b568/Source/Csla.Blazor/PropertyInfo.cs#L33

Reply:
Oh, my bad!